docs/userguide/simulation.md

```diff
@@ -1,4 +1,121 @@
 # Simulation
 
-This page will soon contain information about simulations.
-In the meantime, please see the [examples](../../examples/examples) instead.
\ No newline at end of file
+BayBE offers multiple functionalities to "simulate" experimental campaigns with a given lookup mechanism. This user guide briefly introduces how to use the methods available in our [simulation submodule](baybe.simulation).
+
+For a wide variety of applications of this functionality, we refer to the corresponding [examples](../../examples/Backtesting/Backtesting).
+
+## Terminology: What do we mean by "Simulation"?
+
+The term "simulation" can have two slightly different interpretations, depending on the applied context.
+
+1. It can refer to "backtesting" a particular experimental campaign on a fixed finite dataset.
+In this context, "simulation" means investigating what experimental trajectory we would have observed if we had applied the recommender in a certain defined context and restricted the possible parameter configurations to those contained in the dataset.
+
+2. It can refer to the simulation of an *actual* DOE loop, i.e., recommending experiments and retrieving the corresponding measurements, where the loop closure is realized in the form of a callable (black-box) function that can be queried during the optimization to provide target values.
+
+In BayBE, the simulation submodule allows a wide range of use cases and can even be used for "oracle predictions".
```

Comment:
feels like this last paragraph should already be written below the next subheading `The lookup functionality`

Reply:
Which paragraph exactly do you mean here? The one starting with "In BayBE"?

Reply:
yes, the last paragraph before the mentioned heading

Reply:
Changed.